src/filters.py

```diff
@@ -0,0 +1,20 @@
+from api_types import Record, FieldMetadata
+
+
+FIELDS_TO_SKIP = ["date"]
```

Comment:
This is a super minor thing, but this could probably be a `set`. Since we're doing an inclusion check below (`x in y`) that will make that check faster:
```suggestion
FIELDS_TO_SKIP = {"date"}
```